src/openforms/prefill/tests/test_prefill_hook.py

```diff
@@ -231,6 +231,14 @@ def test_applying_prefill_plugins(self):
         self.assertIsNotNone(field["defaultValue"])
         self.assertIsInstance(field["defaultValue"], str)
 
+        with self.subTest("Log event display (gh-5188)"):
```

Comment:
Why did we add this as a subtest?

Reply:
I didn't really understand what/how it was supposed to work so I found an existing test and then checked out older branches to figure out the correct output. From there I was able to piece back how it's supposed to work and what exactly it's supposed to display, so I added some extra logging-specific test code to a general "happy flow" test case.

Then I could figure out what the bug was